classes/PodsInit.php

```diff
@@ -822,6 +824,28 @@ public function register_assets() {
 
 		wp_set_script_translations( 'pods-dfv', 'pods' );
 
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
+			'currencies'     => PodsField_Currency::$currencies,
+			'datetime'       => [
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'datetime'" and double arrow, but found 7.
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: 64752c06f1247fe8563304ebb1899e3d</sub>